gfx/NeoDays/pngs_tiles_10x10/mod/magiclysm/monsters/goblins/credits.txt

```diff
@@ -0,0 +1 @@
+phaethonfire
```

Comment:
don't need credit.txt if you're adding the sprite yourself, the credits will be in the commit history

Reply:
sounds good.  I removed those files now.